diff --git a/src/backend/tty.rs b/src/backend/tty.rs
index 5c8c5b13..91b0eb28 100644
--- a/src/backend/tty.rs
+++ b/src/backend/tty.rs
@@ -786,17 +786,18 @@ impl Tty {
             GbmDevice::new(device_fd)
         }?;
 
-        let mut try_initialize_gpu = || {
+        let mut try_initialize_gpu = || -> anyhow::Result<DrmNode> {
             let display = unsafe { EGLDisplay::new(gbm.clone())? };
             let egl_device = EGLDevice::device_for_display(&display)?;
 
             // Software EGL devices (e.g., llvmpipe/softpipe) are rejected for now. They have some
             // problems (segfault on importing dmabufs from other renderers) and need to be
             // excluded from some places like DRM leasing.
-            ensure!(
-                !egl_device.is_software(),
-                "software EGL renderers are skipped"
-            );
+            // Ensure disabled by user for testing software/VM rendering.
+            // ensure!(
+            //     !egl_device.is_software(),
+            //     "software EGL renderers are skipped"
+            // );
 
             let render_node = egl_device
                 .try_get_render_node()
