diff --git a/Cargo.toml b/Cargo.toml
index 37dee35..0c56315 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -42,13 +42,13 @@ rustdoc-args = [
 
 [package.metadata.system-deps.libdisplay-info]
 name = "libdisplay-info"
-version = ">= 0.1.0, < 0.4.0"
+version = ">= 0.1.0, < 0.5.0"
 
 [package.metadata.system-deps.libdisplay-info.v0_2]
 version = "0.2.0"
 
 [package.metadata.system-deps.libdisplay-info.v0_3]
-version = "0.3.0"
+version = ">= 0.3.0"
 
 [features]
 auto = []
