--- a/plugins/assistants/Assistants/AssistantsToolOptions.ui	2026-09-30 22:54:34.368470600 +0800
+++ b/plugins/assistants/Assistants/AssistantsToolOptions.ui	2026-09-30 22:54:34.370004577 +0800
@@ -284,7 +284,7 @@
       </layout>
      </item>
    <item>
-    <widget class=" QWidget" name="showDockerOptionsPanel">
+    <widget class="QWidget" name="showDockerOptionsPanel">
       <property name="visible">
           <bool>false</bool>
         </property>
